spec/eof.md

```diff
@@ -128,27 +128,21 @@ Code executing within an EOF environment will behave differently than legacy cod
 
 #### Creation transactions
 
-Creation transactions (tranactions with empty `to`), with `data` containing EOF code (starting with `EF00` magic) are interpreted as having a concatenation of EOF `initcontainer` and `calldata` in the `data` and:
+Creation transactions (tranactions with empty `to`), with `data` containing EOF code (starting with `EF00` magic) are interpreted as having a concatenation of an EOF `txcontainer` and `calldata` in the `data` and:
```

Comment:
I invented a new name for this, calling it "an EOF runtime container" seemed somewhat off (?) not sure why